src/pal/src/cruntime/printfcpp.cpp

```diff
@@ -1145,6 +1146,8 @@ int CoreVfwprintf(CPalThread *pthrCurrent, PAL_FILE *stream, const wchar_16 *for
     PERF_ENTRY(vfwprintf);
     ENTRY("vfwprintf (stream=%p, format=%p (%S))\n",
           stream, format, format);
+    
+    va_copy(ap, aparg);
```

Comment:
Ok, I've read some more information on the internet and it now makes sense to me.